js/compressed-token/src/instructions/pack-compressed-token-accounts.ts

```diff
@@ -20,6 +20,11 @@ export type PackCompressedTokenAccountsParams = {
     outputStateTrees?: PublicKey[] | PublicKey;
     /** Optional remaining accounts to append to */
     remainingAccounts?: PublicKey[];
+    /** Root indices that are used onchain to fetch the correct root
+     *  from the state Merkle tree account for validity proof verification.
+     */
```

Comment:
```suggestion
    /**
     *  Root indices that are used on-chain to fetch the correct root
     *  from the state Merkle tree account for validity proof verification.
     */
```

and is fetched from the Merkle tree account directly or from the root history array?

Reply:
from the root history array which is in the Merkle tree account